index.js

```diff
@@ -82,6 +82,9 @@ exports.register = function (server, options, next) {
                 }, function(err, result){
 
                     if(err){
+                        if (debug) {
+                            internals.log('error', err.formatted);
```

Comment:
Ah yes. updating `node-sass@3.4.2` fixes the issue for me. It would seem that's an undocumented property. 

I'm going to keep the dependency range the same (as it's still valid) and just add a check for the existence of that property before using it. Something like:

```js
if (debug) {
  let message = err.formatted ? err.formatted : err.message;
  internals.log('Compilation failed: %s', message);
}
```

Sound good? I'll make these changes in `v2.1.0`